### PR TITLE
Kuma Report, September 2017

### PR DESCRIPTION
**Pull request by jwhitlock**
Here's what happened in September:
- Ran Maintenance Mode Tests in AWS (@metadave, @escattone)
- Updated Article Styling (@stephaniehobson)
- Continued Conversion to Browser Compat Data (@Elchi3, @teoli2003, @wbamberg)
- Shipped Tweaks and Fixes

Here's the plan for October:
- Move MDN to AWS (@metadave, @escattone)
- Improve Performance of the Interactive Editor (@wbamberg, @schalkneethling)

### CODE REVIEW

**Comment by metadave** [comment]
+1, nice writeup of the mm test!

**Comment by Elchi3** [comment]
Excellent report, John.

What the browser compat data project is proud of in September:
- Finished compat data migration of all HTML and JavaScript pages.
- More than 1,500 MDN pages are now generating the compat tables from the BCD data (about 4,500 are still using static tables).
- The repository reached 50 contributors.

File: _posts/2017-10-05-kuma-report.md [hunk 1]
```diff
@@ -0,0 +1,222 @@
...
+Before PR 4400, a fixed-width font was used to display literals:
+
+![Before 4400 no highlight]({{ site.baseurl }}/public/images/kuma/2017-09-before-4400.png "Before 4400")
+
+After PR 4000, the literals stand out with a light grey background:
+
+![After 4400 highlight]({{ site.baseurl }}/public/images/kuma/2017-09-after-4400.png "After 4400")
+
+There's a lot that goes into making text on the web readable (see
+[Stephanie's slides](https://www.slideshare.net/stephaniehobson/writing-for-every-reader)
```
Comment (stephaniehobson) [inline _posts/2017-10-05-kuma-report.md]: ❤️

**Comment by escattone** [comment]
👍 Nice work as always @jwhitlock , thanks!

**Comment by jwhitlock** [comment]
Thanks for the reviews! Publishing...

**Pull request merged**